### DISCUSSION TITLE
Experiment naming convention

### DISCUSSION THREAD

**Discussion by mvertens** [NorESMhub/noresm3_dev_simulations#16] [General]
My understanding from the meeting is that we proposed an experment naming convent of 
`compset_alias.codeversion.grid_alias..somekindofinfo.yyyyymmdd`
But I think its awkward to place the codversion inbetween the compset and the grid - so I would propose an experimental naming convention of
`compset_alias.grid_alias.codeversion.somekindofinfo.yyyyymmdd`
Is there any disagreement with this?

**Comment by adagj** [comment]
I think that sounds like a good idea. Thanks!
